### PR TITLE
Tests for TG-244

### PR DESCRIPTION
**Pull request by sergii-trotsiuk**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/142" target="_blank">ReviewNinja

**Comment by velmyk** [comment]
Can you please move most of preparations to beforeEach section. For 'it' to contain only assertion we check

File: src/app/core/auth/auth.service.test.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    return this;
+                }
+            };
+            shHttpMock.post.and.returnValue(httpPostObservable);
+
+            // Run
+            const result = service.login(credentials);
+
+            // Expect
+            expect(result).toBe(httpPostObservable);
```
Comment (velmyk) [inline src/app/core/auth/auth.service.test.ts:40]: Please check one assertion per test

Comment (sergii-trotsiuk) [inline src/app/core/auth/auth.service.test.ts:40]: Test for - expect(result).toBe(httpPostObservable);
Test for - expect(shHttpMock.post).toHaveBeenCalledWith(authApiUrl, credentials);
And test for - expect(profileMock.setUserData).toHaveBeenCalledWith(authLoginPostData.user, authLoginPostData.token);

?

Comment (sergii-trotsiuk) [inline src/app/core/auth/auth.service.test.ts:40]: It's one case. So I think few assertion in this test is ok.

**Comment by sergii-trotsiuk** [comment]
@velmyk

> Can you please move most of preparations to beforeEach section. For 'it' to contain only assertion we check

Could you specify code that you think better move to beforeEach?

File: src/app/core/auth/auth.service.test.ts (hunk 1 above)
Comment (velmyk) [inline src/app/core/auth/auth.service.test.ts:40]: Different assertion related to one case can be grouped with describe section. But it is better when one assertion can fail only one test

**Pull request merged**